modules/stochastic_tools/include/utils/Standardizer.h

```diff
@@ -0,0 +1,62 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#pragma once
+
+#include "MooseTypes.h"
+#include <vector>
+#include "DataIO.h"
+
+namespace StochasticTools
+{
+
+/* Class standardizing data (centering and scaling)
+
+ */
+class Standardizer
+{
+public:
+  Standardizer(){};
+
+  /// Methods for setting mean and standard deviation directly
+  /// Sets mean=0, std=1 for n variables
+  void set(const Real & n);
+  /// Sets mean and std for a single variable
+  void set(const Real & mean, const Real & stdev);
+  /// Sets mean and std for a n variables variable
+  void set(const Real & mean, const Real & stdev, const Real & n);
+  /// Sets mean and std directly using provided vectors
+  void set(const std::vector<Real> & mean, const std::vector<Real> & stdev);
+
+  /// Methods for computing and setting mean and standard
+  void computeSet(const RealEigenMatrix & input);
+
+  /// Helper for dataStore
+  void storeHelper(std::ostream & stream, void * context) const;
+
+  /// Returns the standardized (centered and scaled) of the provided input
+  RealEigenMatrix getStandardized(const RealEigenMatrix & input) const;
+
+  /// De-standardizes (de-centered and de-scaled) the assumed standardized input
+  RealEigenMatrix getDestandardized(const RealEigenMatrix & input) const;
+
+  /// De-scales the assumed scaled input
+  RealEigenMatrix getDescaled(const RealEigenMatrix & input) const;
```

Comment:
Done. It is a slight conceptual change, since it is conceivable (but not in use) that a parson may want the standardized data AND the original data for some use. In this case they would just need to copy the data before standardizing one of the copies.